<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4IQHI62" name="EcomProduct Testscripts - Sanity" tests="18" failures="6" timestamp="2 Dec 2016 10:11:16 GMT" time="99.521" errors="0">
  <testcase name="createProduct" time="11.121" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="createProduct" time="11.842" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="createProduct" time="5.783" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="createProduct" time="4.419" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="createProduct" time="10.0" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="deleteProduct" time="3.456" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.lang.AssertionError" message="Expected to throw 204 status, but it thrown: 200 Status Message: OK expected [204] but found [200]">
      <![CDATA[java.lang.AssertionError: Expected to throw 204 status, but it thrown: 200 Status Message: OK expected [204] but found [200]
	at com.wooplr.consumers.EcomProductConsumer.getProduct(EcomProductConsumer.java:84)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.deleteProduct(EcomProductTestPlan.java:137)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- deleteProduct -->
  <testcase name="deleteProduct" time="5.058" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.lang.AssertionError" message="Expected to throw 204 status, but it thrown: 200 Status Message: OK expected [204] but found [200]">
      <![CDATA[java.lang.AssertionError: Expected to throw 204 status, but it thrown: 200 Status Message: OK expected [204] but found [200]
	at com.wooplr.consumers.EcomProductConsumer.getProduct(EcomProductConsumer.java:84)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.deleteProduct(EcomProductTestPlan.java:137)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- deleteProduct -->
  <testcase name="getByInvalidProductId" time="0.79" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.lang.AssertionError" message="Expected to throw 204 status, but it thrown: 500 Status Message: Internal Server Error expected [204] but found [500]">
      <![CDATA[java.lang.AssertionError: Expected to throw 204 status, but it thrown: 500 Status Message: Internal Server Error expected [204] but found [500]
	at com.wooplr.consumers.EcomProductConsumer.getProduct(EcomProductConsumer.java:84)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.getByInvalidProductId(EcomProductTestPlan.java:93)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- getByInvalidProductId -->
  <testcase name="getByInvalidProductId" time="0.868" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.lang.AssertionError" message="Expected to throw 204 status, but it thrown: 500 Status Message: Internal Server Error expected [204] but found [500]">
      <![CDATA[java.lang.AssertionError: Expected to throw 204 status, but it thrown: 500 Status Message: Internal Server Error expected [204] but found [500]
	at com.wooplr.consumers.EcomProductConsumer.getProduct(EcomProductConsumer.java:84)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.getByInvalidProductId(EcomProductTestPlan.java:93)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- getByInvalidProductId -->
  <testcase name="getProduct" time="3.381" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="4.917" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="10.987" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="4.681" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="6.211" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="5.141" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateProduct" time="7.269" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan"/>
  <testcase name="updateSkuDetails" time="1.694" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.util.ConcurrentModificationException">
      <![CDATA[java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.checkForComodification(Unknown Source)
	at java.util.ArrayList$Itr.next(Unknown Source)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.updateSkuDetails(EcomProductTestPlan.java:153)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- updateSkuDetails -->
  <testcase name="updateSkuDetails" time="1.44" classname="com.wooplr.ecomproduct.test.EcomProductTestPlan">
    <failure type="java.util.ConcurrentModificationException">
      <![CDATA[java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.checkForComodification(Unknown Source)
	at java.util.ArrayList$Itr.next(Unknown Source)
	at com.wooplr.ecomproduct.test.EcomProductTestPlan.updateSkuDetails(EcomProductTestPlan.java:153)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- updateSkuDetails -->
</testsuite> <!-- EcomProduct Testscripts - Sanity -->
